tests/phpunit/integration/PluginTest.php

```diff
@@ -43,6 +43,14 @@ public function test_context() {
 		$this->assertEquals( plugin_dir_path( GOOGLESITEKIT_PLUGIN_MAIN_FILE ), $plugin->context()->path() );
 	}
 
+	public function test_plugin_data() {
+		// Ensure we're supplying the correct minimum version of PHP
+		// and WordPress in our plugin file's header.
+		$this->assertEquals( get_plugin_data( GOOGLESITEKIT_PLUGIN_MAIN_FILE )['RequiresPHP'], '5.6' );
```

Comment:
Unfortunately, `RequiresPHP` wasn't added until 5.3 so this fails on our job using the minimum versions of PHP and WP.

https://github.com/google/site-kit-wp/runs/6260212583?check_suite_focus=true#step:10:44

Let's conditionally make this assertion based on the current WP version.

Reply:
Argh, whoops, I only tested locally with PHP 7. Will-do, thanks! 👍🏻